tests/resolve_table.c

```diff
@@ -20,7 +20,19 @@ without any warranty. */
     if (!(test))      \
       result = 1;     \
   } while(0)          \
-    
+
+
+static void setTablePath(const char *value) {
```

Comment:
We should maybe explore the use of the [gnulib setenv module](http://git.savannah.gnu.org/gitweb/?p=gnulib.git;a=blob_plain;f=modules/setenv) which would hide that kind of `#ifdef` stuff away